package.json

```diff
@@ -19,10 +19,12 @@
   ],
   "files": [
     "dist/",
+    "index.d.ts",
```

Comment:
I think if the `typings` field points to the right place it shouldn't matter, but I'll test everything out.

I wanted to avoid copying, as `cp` is *nix-specific, and I wanted to avoid adding a dependency like `ncp` just for that task.